news/_posts/translation/2022-09-16-do-you-speak-prestashop-august-2022.md

```diff
@@ -0,0 +1,130 @@
+
+### 🛠️ Working on resolving some issues
+
+You may have noticed that translation updates are currently not available in PrestaShop back office. Our team is working on resolving the issue, we will keep you posted on the situation!
+
+## A few stats
+ 
+* **55 members** were active on the project this month.
+* A total of **57,817 words have been translated** and **44,940 validated**.
+* All this in **30 different languages**.
+ 
+Thank you very much for your involvement!
+
+## Top contributors
+ 
+A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and PrestaShop cannot thank you enough for your dedication! Here are the most active translators and proofreaders in August 2022.
+ 
+#### Top 10 translators in August:
+ 
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+| 1. | danielshapiro9 | Hebrew | 10,071
+| 2. | Daniel Hlavacek (Hlavtox) | Czech | 7,396
+| 3. | webdvl (megashopba) | Slovak, Czech | 5,650
+| 4. | Enzo Ferrara (ferribbotte) | Italian | 4,112
+| 5. | Michał Boruta (eveneme) | Polish | 3,943
+| 6. | Ramon J. Batlle (rjbatllet) | Catalan | 3,014
+| 7. | SeongHyeon Cho (jaymz9634) | Korean | 2,957
+| 8. | Rodrigo Laurindo (rblaurin) | Portuguese, Brazilian | 2,952
+| 9. | Monika (monikaraciunaite) | Lithuanian | 2,821
+| 10. | diesserr | Catalan, Spanish | 1,603
+
+
+#### Top 5 proofreaders in August:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | danielshapiro9 | Hebrew | 9,341
+| 2. | webdvl (megashopba) | Slovak, Czech | 7,883
+| 3. | Daniel Hlavacek (Hlavtox) | Czech | 7,416
+| 4. | Benjamin Gantikow (bbbenjie) | German | 3,448
+| 5. | Ramon J. Batlle (rjbatllet) | Catalan | 3,415
+| 6. | Rodrigo Laurindo (rblaurin) | Portuguese, Brazilian | 2,963
+| 7. | SeongHyeon Cho (jaymz9634) | Korean | 2,947
+| 8. | Monika (monikaraciunaite) | Lithuanian | 2,906
+| 9. | Rauno Riikman (weaver) | Finnish | 1,770
+| 10. | Enzo Ferrara (ferribbotte) | Italian | 776
+
+
+Congratulations, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who has been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
+ 
+## Complete translations
+ 
+#### Fully available languages
+ 
+At the end of August, PrestaShop 1.7.8 was fully available (= 100% translated and validated) in **14 languages**. That’s great! Thank you for your involvement. :tada:
+ 
+| Bosnian | Catalan | Czech | Chinese (CN) |
+| Chinese (TW) | Danish | English | Estonian | 
+| Finnish | French | Hungarian | Italian |
+| Lithuanian | Portuguese, BR | **Slovak** | 
+
+Thank you all for your contribution! :muscle: 
+
+#### Almost there!
```

Comment:
I think we can remove this section (Almost there!) after all. Because we don't want translators to work on 1.7.8 strings anymore and we already encouraged them to work on the PrestaShop 8 strings at the beginning of the article.